languages/storefront.pot

```diff
@@ -4,7 +4,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Storefront 2.6.0\n"
 "Report-Msgid-Bugs-To: https://github.com/woothemes/storefront/issues\n"
-"POT-Creation-Date: 2020-08-04 17:11:42+00:00\n"
+"POT-Creation-Date: 2020-09-18 13:57:57+00:00\n"
```

Comment:
You'll want to remove these changes from this PR !

Reply:
The POT file had been removed.